src/citrine/seeding/find_or_create.py

```diff
@@ -91,3 +93,32 @@ def default_provider():
             return dataset_collection.register(Dataset(dataset_name, "seed summ.", "seed desc."))
         dataset = get_by_name_or_create(dataset_collection, dataset_name, default_provider)
     return dataset
+
+
+def create_or_update(collection: Collection,
+                     resource: CreationType)->CreationType:
+    """Update a resource of a given name belonging to a collection.
+    Create it if it doesn't exist. 
+    Parameters
+    ----------
+    collection : Collection
+        Collection within which you want to update or create a resource
+    resource : CreationType
+        Resource that you want to create or update. If collection already contains
+        a resource with the same name it will be updated. Otherwise it will be created.
+    Returns
+    -------
+    ResourceType
+        Registered updated or created resource.
+    """    
+    try:
+        old_resource = get_by_name_or_raise_error(collection, resource.name)
+        print(f"Updating module: {resource.name}")
```

Comment:
f-strings aren't supported in python 3.5, which we do support.  I suggest using the `format` method instead.

Reply:
I changed this in the newest commits.